source/scripting_v3/GTA/Camera/CinematicCameraDirector.cs

```diff
@@ -0,0 +1,126 @@
+		/// Only available in v1.0.372.2 or later versions.
+		/// </summary>
+		/// <value>
+		/// <see langword="true" /> if the <see cref="CinematicCameraDirector"/> is a vehicle nose camera; otherwise, <see langword="false" />.
+		/// </value>
+		/// <exception cref="GameVersionNotSupportedException">
+		/// Thrown if called in game versions earlier than v1.0.372.2.
+		/// </exception>
+		public static bool IsVehicleBonnetCamRendering
+		{
+			get
+			{
+				GameVersionNotSupportedException.ThrowIfNotSupported(GameVersion.v1_0_372_2_Steam, nameof(CinematicCameraDirector), nameof(IsVehicleBonnetCamRendering));
+
+				return Function.Call<bool>(Hash.IS_BONNET_CINEMATIC_CAM_RENDERING);
+			}
+		}
+
+		/// <summary>
+		/// Gets a value indicating whether any cinematic camera is rendering.
+		/// Note that this will also return true if in first-person view while inside a vehicle.
+		/// </summary>
+		/// <value>
+		/// <see langword="true" /> if the <see cref="CinematicCameraDirector"/> is a cinematic camera; otherwise, <see langword="false" />.
+		/// </value>
+		public static bool IsAnyCamRendering => Function.Call<bool>(Hash.IS_CINEMATIC_CAM_RENDERING);
+
+		/// <summary>
+		/// Gets a value indicating whether an idle cinematic camera is rendering.
+		/// </summary>
+		/// <value>
+		/// <see langword="true" /> if the <see cref="CinematicCameraDirector"/> is an idle cinematic camera; otherwise, <see langword="false" />.
+		/// </value>
+		public static bool IsIdleCamRendering => Function.Call<bool>(Hash.IS_CINEMATIC_IDLE_CAM_RENDERING);
+
+		/// <summary>
+		/// Gets a value indicating whether a player-controlled cinematic camera is active.
+		/// Only available in v1.0.1493.0 or later game versions.
+		/// </summary>
+		/// <value>
+		/// <see langword="true" /> if the <see cref="CinematicCameraDirector"/> is a player-controlled cinematic camera; otherwise, <see langword="false" />.
+		/// </value>
+		/// <exception cref="GameVersionNotSupportedException">
+		/// Thrown if called in game versions earlier than v1.0.1493.0.
+		/// </exception>
+		public static bool IsPlayerCamActive
+		{
+			get
+			{
+				GameVersionNotSupportedException.ThrowIfNotSupported(GameVersion.v1_0_1493_0_Steam, nameof(CinematicCameraDirector), nameof(IsPlayerCamActive));
+
+				return Function.Call<bool>(Hash.IS_CINEMATIC_CAM_INPUT_ACTIVE);
+			}
+		}
+
+		/// <summary>
+		/// Shakes the <see cref="CinematicCameraDirector"/>'s active camera.
+		/// </summary>
+		/// <param name="shakeType">Type of the shake to apply.</param>
+		/// <param name="amplitude">The amplitude of the shaking.</param>
+		public static void Shake(CameraShake shakeType, float amplitude)
+		{
+			Function.Call(Hash.SHAKE_CINEMATIC_CAM, Camera.s_shakeNames[(int)shakeType], amplitude);
+		}
+
+		/// <summary>
+		/// Stops shaking the <see cref="CinematicCameraDirector"/>'s active camera.
+		/// </summary>
+		public static void StopShaking()
+		{
+			Function.Call(Hash.STOP_CINEMATIC_CAM_SHAKING, true);
```

Comment:
I think you probably didn't have the idea what the bool param represents as alloc8or's Native DB doesn't have the info, but this is the same as `STOP_CAM_SHAKING` which is `bStopImmediately`. Lets change the signature to `StopShaking(bool stopImmediately = false)` (that param is set to false by default in that scm header).

I know `GameplayCamera.StopShaking()` implicitly specifies the bool param false, but we weren't sure what that bool param means when v3.0 was released.